Source/WebCore/Modules/mediastream/UserMediaController.cpp

```diff
@@ -45,14 +45,15 @@ ASCIILiteral UserMediaController::supplementName()
     return "UserMediaController"_s;
 }
 
-UserMediaController::UserMediaController(UserMediaClient* client)
+UserMediaController::UserMediaController(const RefPtr<UserMediaClient>& client)
     : m_client(client)
 {
 }
 
 UserMediaController::~UserMediaController()
 {
-    m_client->pageDestroyed();
+    if (RefPtr<UserMediaClient> mediaClient = m_client.get())
```

Comment:
```suggestion
    if (RefPtr mediaClient = m_client.get())
```
let template deduction work.